handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -321,6 +321,19 @@ private void replaceHandler(ChannelHandlerContext ctx, Selection selection) {
         }
     }
 
+    /**
+     * Users may override this method to implement custom behavior. The default
+     * implementation will simply replace {@code this} {@link SniHandler} instance
+     * with the given {@link SslHandler} using the provided {@link SSLEngine}.
+     *
+     * BEWARE: The {@code hostname} argument can be {@code null}.
+     */
+    protected void replaceHandler(ChannelHandlerContext ctx, String hostname,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "hostname". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)